SECURITY.md

```diff
@@ -6,7 +6,12 @@ Patches for **Rokwire Building Blocks** in this repository will only be applied
 
 | Version  | Supported          |
 |----------| ------------------ |
+<<<<<<< HEAD
+| 1.16.0   | ✅ |
```

Comment:
Please update the security.MD to use the following for the first two versions:
```
| 1.16.0   | :white_check_mark: |
| 1.15.0   | :white_check_mark: |
```
It looks like currently, it's not using markdown code. Maybe my suggestion automatically converted it to an icon? Sorry if that's the case.

Also, please remove the markers from the merge conflict. Thanks.